pkg/resource/prunercronjob.go

```diff
@@ -239,7 +239,17 @@ func (gcj *generatorPrunerCronJob) getKeepYoungerThan(cr *imageregistryapiv1.Ima
 }
 
 func (gcj *generatorPrunerCronJob) getLogLevel(cr *imageregistryapiv1.ImagePruner) int {
-	return loglevel.LogLevelToVerbosity(cr.Spec.LogLevel)
+	level := loglevel.LogLevelToVerbosity(cr.Spec.LogLevel)
```

Comment:
I think it makes it more assertive if we do something like this:

if cr.spec.loglevel == normal {
    return 1
}
return loglevel.LogLevelToVerbosity(cr.Spec.LogLevel)

no further comment would be needed as our normal is level 1. what do you think?

Reply:
Take a look at LogLevelToVerbosity, it returns 2 when it gets Normal or something unknown (like an empty string). The comment will be needed anyway because it's unclear why we cannot use cluster defaults.